submissions/AnnaStoyano/Movie_seats_booking/index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cinema task</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header__inner flex">
+            <a class="logo-link" href="/">
+                <img class="logo-img" src="./images/Road_Movies.png" alt="Cinema logo">
+            </a>
+            <nav class="navigation">
+                <ul class="navigation__list flex">
+                    <li class="navigation__item"><a href="/">Rental films</a></li>
+                    <li class="navigation__item"><a href="/">Soon</a></li>
+                    <li class="navigation__item"><a href="/">News</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <article class="film-poster">
+            <div class="film-info">
+                <h2 class="film-title">
+                    Palace for Putin. The history of the biggest bribe
+                </h2>
+                <p class="film-description">
+
+                    Today you will see what is considered impossible to see up close. Together with us you will go where
+                    no one is allowed. We will visit Putin. Let's make sure with our own eyes that this person is in
+                    his craving for luxury, for wealth, he completely lost his mind. We will find out with whose money
+                    and how this
+                    luxury. And how, over the past 15 years, the biggest bribe in history has been given and the most
+                    dear palace in the world.
+
+                </p>
+                <div class="rating__wrapper">
+                    <img class="star" src="./images/star.svg" alt="rating star icon">
+                    <strong class="raiting">IMDb: 9.8/10</strong>
+                </div>
+            </div>
+            <img class="film_img" src="./images/film_logo.jpg" alt="Putin's Palace">
+        </article>
+
+        <section class="timeslot">
+            <h3 class="timeslot__title title">Time</h3>
+            <ul class="timeslot__list flex">
+                <li class="timeslot__item">
+                    <button name="time" class="time_btn" value="11:00">11:00</button>
```

Comment:
For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)